agents/passports/observability/PASSPORT.md

```
@@ -22,6 +21,50 @@
 - MUST NOT delete health_log or alert_log (I-24)
 - MUST NOT push to Grafana without BT-008 resolved (NotImplementedError)
 
+## HITL Gates
+| Action | Gate | Role |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)